src/components/calcite-radio-button/calcite-radio-button.scss

```diff
@@ -0,0 +1,110 @@
+:host-context([theme="dark"]) {
+  @include calcite-theme-dark();
+}
+:host {
+  display: grid;
+  grid-gap: 8px;
+  align-items: center;
+  cursor: pointer;
+  #radio {
```

Comment:
Sure.  I used `id` mainly because I could (since it's inside Shadow DOM and wouldn't pollute the global ids of the document) but if the pattern is to use classes that's totally doable too.